src/main/kotlin/no/nav/klage/oppgave/domain/kafka/KlageStatistikkTilDVH.kt

```diff
@@ -0,0 +1,136 @@
+
+    @JsonSchemaDescription("Kode som angir den aktuelle behandlingens tilstand på gjeldende tidspunkt. Ha med alle mulige statuser som er naturlig for det enkelte system/ytelse. Som minimum, angi om saken har følgende status: Registrert, Klar for behandling, Venter på bruker, venter på ekstern (arbeidsgiver, lege etc.), venter på utland, Avsluttet.Her bør det også angis at saken er behandlet av beslutter, men sendt i retur for ny behandling.")
+    val behandlingStatus: String,
+
+    @JsonSchemaDescription("Kode som beskriver behandlingen, for eksempel, søknad, revurdering, klage, anke, endring, gjenopptak, tilbakekreving o.l.")
+    val behandlingType: String,
+
+    @JsonSchemaDescription("Bruker IDen til den ansvarlige beslutningstageren for saken. Medunderskriver.")
+    val beslutter: String?,
+
+    @JsonFormat(
+        shape = JsonFormat.Shape.STRING,
+        pattern = DATE_TIME_FORMAT
+    )
+    @JsonSchemaFormat(DATE_TIME_FORMAT)
+    @JsonSchemaDescription("Tidspunktet da hendelsen faktisk ble gjennomført eller registrert i kildesystemet. (format:$DATE_TIME_FORMAT) Dette er det tidspunkt der hendelsen faktisk er gjeldende fra. Ved for eksempel patching av data eller oppdatering tilbake i tid, skal tekniskTid være lik endrings tidspunktet, mens funksjonellTid angir tidspunktet da endringen offisielt gjelder fra.")
+    val endringstid: LocalDateTime,
+
+    @JsonSchemaDescription("Kommaseparert liste av hjemler.")
+    val hjemmel: String,
+
+    @JsonSchemaDescription("Den som sendt inn klagen.")
+    val klager: Part,
+
+    @JsonSchemaDescription("Grunn til utfallet, hvis det finnes.")
+    val omgjoeringsgrunn: String?,
+
+    @JsonSchemaDescription("F.eks. Foreldrepenger. Det finnes kodeverk for dette.")
+    val opprinneligFagsaksystem: String,
+
+    @JsonFormat(
+        shape = JsonFormat.Shape.STRING,
+        pattern = DATE_FORMAT
+    )
+    @JsonSchemaFormat(DATE_FORMAT)
+    @JsonSchemaDescription("Når KA mottok oversendelsen.")
+    val overfoertKA: LocalDate,
+
+    @JsonSchemaDescription("Utfall.")
+    val resultat: String?,
+
+    //Hvis EØS kommer tilbake så legg til dette.
+//    /**
+//    "maxLength": 100 ,
+//    "description": "Kode som beskriver behandlingens utlandstilsnitt i henhold til NAV spesialisering. I hoved sak vil denne koden beskrive om saksbehandlingsfrister er i henhold til utlandssaker eller innlandssaker, men vil for mange kildesystem være angitt med en høyere oppløsning."
+//     */
+//    val utenlandstilsnitt: String,
+
+    @JsonSchemaDescription("Den som har rettigheten.")
+    val sakenGjelder: Part,
+
+    @JsonSchemaDescription("Bruker IDen til saksbehandler ansvarlig for saken på gjeldende tidspunkt. Kan etterlates tom ved helautomatiske delprosesser i behandlingen. Bør bare fylles når det er manuelle skritt i saksbehandlingen som utføres.")
+    val saksbehandler: String?,
+
+    @JsonSchemaDescription("Enhet til gjeldende saksbehandler.")
+    val saksbehandlerEnhet: String?,
```

Comment:
Hva er forskjellen på denne og ansvarligEnhetKode? Er ansvarligEnhetKode førsteinstans?

Reply:
Jeg tror `ansvarligEnhetKode` egentlig var om en gitt enhet var tildelt/ansvarlig for klagen, selv uten å være tildelt en saksbehandler. Når vi nå skal gå for "nasjonal kø", så gir ikke det så mye mening.

Reply:
Men det der - "om en gitt enhet var tildelt klagen, selv uten å være tildelt en saksbehandler", det er jo det vi har brukt tildeltEnhet til i Klagebehandling. Vi har jo bare 1 enhet vi, mens i denne klassen har du to?

Reply:
Ja, vi bør nok ikke fylle ut `ansvarligEnhetKode` per nå tenker jeg.